index.js

```diff
@@ -121,7 +121,10 @@ class DockerExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const [buildImage, buildTag] = config.container.split(':');
+        const containerName = config.container.split(':');
+
+        const buildTag = (containerName.length === 1) ? 'latest' : containerName.pop();
+        const buildImage = containerName.join(':');
```

Comment:
You might need to use regexp to switch the parsing logic.
This module would be helpful in this case: https://www.npmjs.com/package/docker-parse-image